test/expected/insert-11.out

```diff
@@ -550,3 +550,38 @@ EXPLAIN (costs off) INSERT INTO date_inf SELECT * FROM date_inf
                                  Index Cond: ("time" > '-infinity'::date)
 (9 rows)
 
+-- test INSERT with cached plans / plpgsql functions
+-- https://github.com/timescale/timescaledb/issues/1809
+CREATE TABLE status_table(a int, b int, last_ts timestamptz, UNIQUE(a,b));
+CREATE TABLE metrics(time timestamptz NOT NULL, value float);
+CREATE TABLE metrics2(time timestamptz NOT NULL, value float);
+SELECT (create_hypertable(t,'time')).table_name FROM (VALUES ('metrics'),('metrics2')) v(t);
+ table_name 
+------------
+ metrics
+ metrics2
+(2 rows)
+
+INSERT INTO metrics VALUES ('2000-01-01',random()), ('2000-02-01',random()), ('2000-03-01',random());
+CREATE OR REPLACE FUNCTION insert_test() RETURNS VOID LANGUAGE plpgsql AS $$
```

Comment:
Nit: would be nice to have the start `$$` marker at the same level/indentation as the end one.